lib/benchpark/cmd/info.py

```diff
@@ -0,0 +1,195 @@
+            for resource_key, resource_value in resource_dict.items():
+                if resource_key == "hardware_key":
+                    with open(resource_value, "r") as f:
+                        data = yaml.safe_load(f)
+                        colors = ["@*r", "@*c", "@*m", "@*y"]
+                        data = _replace_keys_with_colors(data, colors)
+                        color.cprint(
+                            textwrap.indent(
+                                yaml.dump(data).replace("'", ""), indent() * 2
+                            )
+                        )
+                else:
+                    resource_key = "@*r" + resource_key + "@."
+                    color.cprint(f"{indent()*2}{resource_key}: {resource_value}")
+
+    system_spec = benchpark.spec.SystemSpec(args.name)
+    system_class = system_spec.system_class
+
+    # Map argument flags to functions
+    actions = {
+        "hardware": (_info_system_hardware, [system_class]),
+        "maintainers": (info_maintainers, [system_class]),
+        "system_site": (_info_system_system_site, [system_class]),
+        "variants": (info_variants, [system_class]),
+    }
+
+    # Call functions for enabled options, or all if no flag is set
+    any_flag_set = False
+    for flag, (action, aargs) in actions.items():
+        if getattr(args, flag, False):
+            action(*aargs)
+            any_flag_set = True
+
+    if not any_flag_set:
+        for action, aargs in actions.values():
+            action(*aargs)
+
+
+def info_experiment(args):
+    def _info_url(experiment_class):
+        gen_header("URL")
+        print(indent() + getattr(experiment_class, "url", "None"))
+
+    def _info_spack_name(experiment_class):
+        gen_header("Spack Name")
+        print(
+            indent()
+            + (
+                experiment_class.spack_name
+                if experiment_class.spack_name
+                else experiment_class.name
+            )
+        )
+
+    def _info_ramble_name(experiment_class):
+        gen_header("Ramble Name")
+        print(
+            indent()
+            + (
+                experiment_class.ramble_name
+                if experiment_class.ramble_name
+                else experiment_class.name
+            )
+        )
+
+    experiment_spec = benchpark.spec.ExperimentSpec(args.name)
+    conc = experiment_spec.concretize()
+    experiment_class = conc.experiment
+
+    if args.spack:
+        subprocess.run(["spack", "info", experiment_class.spack_name])
+        return
+    elif args.ramble:
+        subprocess.run(["ramble", "info", experiment_class.ramble_name])
+        return
+    else:
+        actions = {
+            "maintainers": (info_maintainers, [experiment_class]),
+            "ramble_name": (_info_ramble_name, [experiment_class]),
+            "spack_name": (_info_spack_name, [experiment_class]),
+            "url": (_info_url, [experiment_class]),
```

Comment:
We should not be keeping a URL in experiment.py since package.py already keeps a git URL (and in some cases the homepage).  We do need to be able to query the git URL, and the version("develop", branch="caliper"), particularly so we can determine the versions we are using, and where we are using specific branches.

Reply:
I can remove the urls. 

I think the following satisfies that query - `benchpark info experiment --spack experiment_name`? e.g.:
![image](https://github.com/user-attachments/assets/a0544abf-d65d-430c-949d-e353c8dff7da)

Reply:
Does ``benchpark info experiment --spack experiment_name`` look at the package.py in benchpark/repo first?

Reply:
I want to make sure we capture the info from ``benchpark/repo/laghos/package.py``::

    homepage = "https://github.com/wdhawkins/laghos"
    git = "https://github.com/wdhawkins/Laghos.git"
    version("develop", branch="caliper")

Reply:
![image](https://github.com/user-attachments/assets/b00f1a12-bf2f-4ed8-b1c5-9a40283ce721)

Currently it does, but only if you have ran `benchpark setup`, i.e. using the spack that is linked to `benchpark/repos/`, which is created on the setup. I assume that this is not an acceptable constraint though, as it makes sense to use `benchpark info` before setting up any experiments.